components/uaa-activator/internal/uaa/waiter.go

```diff
@@ -0,0 +1,57 @@
+package uaa
+
+import (
+	"context"
+
+	"github.com/kyma-project/kyma/components/uaa-activator/internal/repeat"
+
+	"github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	uaaClusterServiceClassName = "xsuaa"
+	uaaClusterServicePlanName  = "z54zhz47zdx5loz51z6z58zhvcdz59-b207b177b40ffd4b314b30635590e0ad"
```

Comment:
How sure are you this plan name will not change soon?

Reply:
We have an agreement that it will not be changed.

Reply:
extracted to env vars, just in case